tests/profiler/test_profiler.py

```diff
@@ -78,7 +78,7 @@ def test_simple_profiler_iterable_durations(simple_profiler, action: str, expect
         pass
 
     # we exclude the last item in the recorded durations since that's when StopIteration is raised
-    np.testing.assert_allclose(simple_profiler.recorded_durations[action][:-1], expected, rtol=0.2)
+    np.testing.assert_allclose(simple_profiler.recorded_durations[action][:-1], expected, rtol=0.25)
```

Comment:
Please keep the PR focused. This is tracked in a separate issue

```suggestion
    np.testing.assert_allclose(simple_profiler.recorded_durations[action][:-1], expected, rtol=0.2)
```

https://github.com/PyTorchLightning/pytorch-lightning/issues/9462